models/web/snowplow_media_plays_by_pageview_this_run.sql

```diff
@@ -0,0 +1,112 @@
+{{
+  config(
+    materialized='table',
```

Comment:
There's no need to define the materialisation as `table` here when you have that in the `dbt_project.yml` file already